.gitignore

```diff
@@ -1,6 +1,10 @@
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# required by nix
+.direnv
+result
```

Comment:
is result a folder? if yes, would you mind put as `result/`

Reply:
done